systers_portal/templates/community/page.html

```diff
@@ -0,0 +1,15 @@
+{% extends "community/base.html" %}
+{% block title %}
+- {{ community }}
+{% endblock %}
+{% block community_page_content %}
+<div class="blog-container">
+<div class="blog-entry">
+<h3 class="title"><a href="#">{{ page.title }}</a></h3>
```

Comment:
Remove the title.
The title of the Community Page is reflected in the Community navigation pills as being highlighted.